tests/unit/test_loadbalance.py

```diff
@@ -77,11 +77,10 @@ def circuit_conf_bigcell():
 
     circuit_base = str(RINGTEST_DIR)
     return CircuitConfig(
-        CircuitPath=circuit_base,
         CellLibraryFile=circuit_base + "/nodes_A_bigcell.h5",
         METypePath=circuit_base + "/hoc",
         MorphologyPath=circuit_base + "/morphologies/asc",
-        nrnPath="<NONE>",  # no connectivity
+        nrnPath=False,  # no connectivity
```

Comment:
it is just slighlty odd that it is set to False and not None for an invalid value

Reply:
Because `nrnPath = ConfigT.REQUIRED` in the definition of class `CircuitConfig` . `None` will raise the error `"Config field {name} is mandatory"`

Reply:
maybe set it to "/this/path/is/not/used", so it's clear that it's just a dummy value, not something magic about `False`?

Reply:
If we needed a hard-coded dummy value, I would revert to the previous `<None>`.

Reply:
I was just wondering why False and not None. Anyway, I do not have hard opinions here. What do you prefer Weina?

Reply:
I will try to revert it to `"NONE"` as before. 
`CircuitConfig` is the internal circuit data structure in neurodamus, with SONATA config files the attributes of `CircuitConfig` are always set properly. It is just here for testing that I kind of hack the path.

Reply:
I think this is a sign that values are being overloaded to mean different things (ie: `"False" to disable the connectivity`), and it should probably be refactored at some point, so that knowledge is more legible.

Reply:
I propose to review and refactor it later. I will create an issue to track that. This PR focuses on deprecating the base/extra_circuit objects.